website/src/components/sidebar.tsx

```diff
@@ -13,33 +18,41 @@ import * as css from "./sidebar.css"
 
 // Renders a sidebar on the left side of the screen containing a drawer.
 export const Sidebar = () => {
+  // On load, make sure drawer is initially visible and is non-modal for desktop screens.
   const dialog = useDialog()
 
-  // On load, make sure drawer appears for desktop screens.
+  // State variable that checks whether this component has loaded in.
+  const [loaded, setLoaded] = useState(false)
+
   useEffect(() => {
     dialog.setVisible(true)
```

Comment:
I had to add this useEffect() back in here because I need this dialog state modified only for desktop screens (have them initially visible and non-modal) and if I use useDialogState() to initiate the visibility and non-modality, I'll still have to add some kind of useEffect for mobile.